From 82700388c42330981a570c898f309558dde786e6 Mon Sep 17 00:00:00 2001
From: Guoqing Jiang <guoqing.jiang@suse.com>
Date: Sat, 7 May 2022 15:36:57 +0800
Subject: [PATCH] Revert "virtio_net: Fix error code in probe()"
Patch-mainline: Never, revert stable patch which is added again in sorted section
References: bsn#198
Modified-by-SEL: No


This reverts commit 78b35fd94cf91d782f6e65ed45f0355215f8fb94.

Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 drivers/net/virtio_net.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 34bb95dd9239..21b71148c532 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -3072,7 +3072,6 @@ static int virtnet_probe(struct virtio_device *vdev)
 			dev_err(&vdev->dev,
 				"device MTU appears to have changed it is now %d < %d",
 				mtu, dev->min_mtu);
-			err = -EINVAL;
 			goto free;
 		}
 
-- 
2.34.1

